zppy/global_time_series.py

```diff
@@ -49,13 +50,18 @@ def global_time_series(config, script_dir, existing_bundles, job_ids_file):
             c["global_time_series_dir"] = os.path.join(script_dir, f"{prefix}_dir")
             if not os.path.exists(c["global_time_series_dir"]):
                 os.mkdir(c["global_time_series_dir"])
-            scripts = ["coupled_global.py", "readTS.py", "ocean_month.py"]
+            scripts = ["coupled_global.py", "ocean_month.py"]
```

Comment:
Contents of `readTS.py` are now in `coupled_global.py`, as mentioned in an earlier comment.